modules/ROOT/pages/intro-configuration.adoc

```diff
@@ -3,23 +3,33 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring.
+In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring. +
 
-These properties are stored in a YAML file:
+Mule 3 supports `.properties` configuration files, while Mule 4 supports both `.yaml` and `.properties`  configuration files. The recommended approach is to use `.yaml` configuration files, because it allows the addition of type validations and autocompletion.
```

Comment:
I'm not sure if the previous rewording suggestions are technically accurate, but if so, you could limit this para to just the one sentence:
```suggestion
Using YAML configuration files enables type validation and autocompletion, which were not possible using Mule 3 `.properties` configuration files.
```